atmo/jobs/models.py

```diff
@@ -204,14 +211,18 @@ def run(self):
             is_public=self.is_public,
             job_timeout=self.job_timeout,
         )
-        self.last_run_date = timezone.now()
+        # Create new job history record.
+        SparkJobHistory.objects.create(
+            spark_job=self,
+            jobflow_id=jobflow_id,
+            run_date = timezone.now(),
```

Comment:
How else could this be handled? Right now it's not much different than what is in `created_at`.